<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="inc_exposure">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>11</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="take_pic">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>11</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Snap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dec_exposure">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_video">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>71</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit Live</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Exposure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_video">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>41</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Live</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="current_exposure">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Y_pos_label">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>470</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="Z_pos_label">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>510</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_knot">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Knot</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="current_Xpos">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="current_Ypos">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>460</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_knot">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Knot</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="current_Zpos">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>500</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Xposition_label">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>430</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_surface">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Surface</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_knots">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Knots</string>
   </property>
  </widget>
  <widget class="GraphicsView" name="knotView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>781</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="GraphicsView" name="gv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="channelsComboBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
